src/taker/offers.rs

```diff
@@ -26,36 +29,33 @@ pub struct OfferAndAddress {
     pub address: MakerAddress,
 }
 
-const REGTEST_MAKER_ADDRESSES: &[&str] = &[
-    "localhost:6102",
-    "localhost:16102",
-    "localhost:26102",
-    "localhost:36102",
-    "localhost:46102",
+const REGTEST_MAKER_ADDRESSES_PORT: &[&str] = &[
+    "6102",
+    "16102",
+    "26102",
+    "36102",
+    "46102",
 ];
 
-/// Enum representing maker addresses (clearnet and TOR).
+/// Enum representing maker addresses.
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub enum MakerAddress {
-    Clearnet { address: String },
-    Tor { address: String },
+    Address(String)
```

Comment:
We don't need the ennum anymore. Just a `MakerAddress(OnionAddres)`?

Reply:
ok.. will change this